src/coreclr/tools/r2rdump/CoreDisTools.cs

```diff
@@ -24,23 +24,22 @@ public enum TargetArch
             Target_Arm64
         };
 
-        [DllImport(_dll)]
+        [DllImport(_dll, CallingConvention = CallingConvention.Cdecl)]
         public static extern IntPtr InitBufferedDisasm(TargetArch Target);
 
-        [DllImport(_dll)]
-        public static extern void DumpCodeBlock(IntPtr Disasm, ulong Address, IntPtr Bytes, int Size);
+        [DllImport(_dll, CallingConvention = CallingConvention.Cdecl)]
+        public static extern void DumpCodeBlock(IntPtr Disasm, IntPtr Address, IntPtr Bytes, int Size);
```

Comment:
`int Size` here corresponds to `size_t Size` in https://github.com/dotnet/jitutils/blob/d9d36de5fb445662b56e583cf00746969f1bfab3/src/coredistools/coredistools.cpp#L816

Shouldn't we update that one as well?

Reply:
Thanks Egor for pointing that out, fixed in 3rd commit.